Formula/picat.rb

```diff
@@ -0,0 +1,19 @@
+class Picat < Formula
+  desc "Simple, and yet powerful, logic-based multi-paradigm programming language"
+  homepage "http://picat-lang.org"
+  url "http://picat-lang.org/download/picat24_src.tar.gz"
+  version "2.4.8"
+  sha256 "72b452a8ba94d6187d837dcdb46aab0d7dc724651bac99a8cf2ada5c0a3543dd"
+
+  def install
+    inreplace "emu/Makefile.picat.mac64", "/usr/local/bin/gcc", "gcc"
+    system "make", "-C", "emu", "-f", "Makefile.picat.mac64"
+    mv "lib", "pi_lib"
+    prefix.install Dir["doc", "emu", "exs", "pi_lib"]
+    bin.install_symlink prefix/"emu/picat_macx" => "picat"
```

Comment:
With that, it seems to me you're installing all the sources and object files in `emu`, which is not desirable.

Reply:
Install picat_macx only from the emu folder.